.yarnrc.yml

```diff
@@ -1,3 +1,10 @@
 nmHoistingLimits: workspaces
 
 nodeLinker: node-modules
+
+plugins:
+  # The plugin source file has to be present in the repository, otherwise it would have to be install
+  # before running yarn install by the user/ci...
+  - .yarn/plugins/plugin-after-install.js
```

Comment:
Should we install the plugin in the workflow or commit it to avoid the [CI failure](https://github.com/getsentry/sentry-react-native/actions/runs/14909879695/job/41881342105?pr=4806)?

Reply:
Ahh, my bad, I've not push the plugin file, when the file is present it should be picked up automatically without installation, which was my goal, as having to install plugin before using `yarn install` would be terrible developer experience.